src/rgw/rgw_op.h

```diff
@@ -1773,7 +1773,8 @@ class RGWGetObjLayout : public RGWOp {
   rgw_raw_obj head_obj;
 
 public:
-  RGWGetObjLayout() {
+  RGWGetObjLayout() = default;
+  ~RGWGetObjLayout() {
```

Comment:
looks correct. how about using `std::unique_ptr<RGWRados::Object> target;` instead though? then we don't need a constructor or destructor at all